share_extension/android/extension_post/extension_post.js

```diff
@@ -227,6 +230,44 @@ export default class ExtensionPost extends PureComponent {
         });
     }
 
+    getAppCredentials = async () => {
+        try {
+            const credentials = await avoidNativeBridge(
+                () => {
+                    return Initialization.credentialsExist;
+                },
+                () => {
+                    return Initialization.credentials;
+                },
+                () => {
+                    return getGenericPassword();
+                }
+            );
+
+            if (credentials) {
+                const passwordParsed = credentials.password.split(',');
+
+                // password == token, url
+                if (passwordParsed.length === 2) {
+                    const [token, url] = passwordParsed;
+
+                    // if for any case the url and the token aren't valid proceed with re-hydration
```

Comment:
I can't find where we actually get the URL and token with rehydration, but that's fine since it'll make it easier when we remove those from the redux state later on